ci/build_cpp.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Update env vars
+source rapids-env-update
+
+# TODO: Move in recipe build?
+export CMAKE_GENERATOR=Ninja
+
+# TODO: Move to job config
+export CUDA=11.5
+
+# Check env
+source ci/check_env.sh
+
+################################################################################
+# BUILD - Conda package builds (LIBRMM)
+################################################################################
+
+gpuci_logger "Begin cpp build"
```

Comment:
We should `grep` this PR for any remnants of `gpuci_*` tools and replace them with the `rapids-*` tools below.

- https://github.com/rapidsai/gha-tools/tree/main/tools